src/main/java/marquez/common/models/SourceQualifier.java

```diff
@@ -14,10 +14,8 @@
 
 package marquez.common.models;
 
-public enum SourceType {
-  MYSQL,
-  POSTGRESQL,
-  REDSHIFT,
-  SNOWFLAKE,
-  KAFKA;
+public enum SourceQualifier {
+  DB,
+  STREAM,
+  FILESYSTEM;
```

Comment:
I would say that the SourceQualifier enum is redundant with the DatasetType enum. We can keep only one. The SourceType can contain datasets of type X (as define by DatasetType). In this PR there is a 1-1 relation between the elements of SourceQualifier and DatasetType. I would suggest to add a field in dataset type for the name of the sourceQualifier if necessary and just remove SourceQualifier altogether.